Cleaned up adfs files

tamigoodall opened the pull request:
Removed excessive MS screenshots
Cleaned up steps, headings, and grammar
Added two troubleshooting cases and fixes

https://docs-content-staging-pr-8037.herokuapp.com/docs/connections/enterprise/adfs
https://docs-content-staging-pr-8037.herokuapp.com/docs/protocols/saml/adfs
https://docs-content-staging-pr-8037.herokuapp.com/docs/protocols/saml/saml-configuration/troubleshoot/auth0-as-sp
https://docs-content-staging-pr-8037.herokuapp.com/docs/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp

<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

tamigoodall committed:
Cleaned up adfs files

Removed excessive MS screenshots
Cleaned up steps, headings, and grammar

tamigoodall committed:
Minor edits

Minor edits

tamigoodall committed:
Clarified steps and updated description

Clarified steps
Updated description in index.md

tamigoodall committed:
Added two troubleshooting cases

Added two troubleshooting cases

tamigoodall committed:
Changed back to ADFS from AD FS

Changed back to ADFS from AD FS

tamigoodall committed:
Fixed broken links

Fixed broken links

tamigoodall committed:
Update auth0-as-sp.md

Clarified text

tamigoodall committed:
Minor tweaks

minor tweaks

tamigoodall committed:
Update adfs.md

Tweaks

tamigoodall committed:
Delete adfs-sign-request-algorithm.png

Deleted unused screenshot

erjohnson commented on articles/connections/enterprise/adfs.md:
```diff
@@ -22,45 +22,41 @@ useCase:
 
 * Realm Identifier: `urn:auth0:${account.tenant}`
-* Endpoint: `https://${account.namespace}/login/callback` (or, if you are using the [custom domains](/custom-domains) feature, `https://<YOUR CUSTOM DOMAIN>/login/callback`)
+* Endpoint: `https://${account.namespace}/login/callback` or `https://<YOUR CUSTOM DOMAIN>/login/callback`, if you are usinga  [custom domain](/custom-domains).
```
using a*

erjohnson commented on articles/protocols/saml/saml-configuration/troubleshoot/common-saml-errors.md:
```diff
@@ -90,3 +90,11 @@ One common error is specifying the incorrect response protocol on the IdP-Initia
+
+## Issue: User isn't logged out from the IdP
+
+When ADFS as configured as SAML IdP, if on the ADFS relaying party trust `Name ID` attribute isn't mapped the logout flow fails. For example, with the federated parameter `v2/logout?federated&...` user isn't redirected to the ADFS SAML logout endpoint but redirects back to application callback URL directly. As a consequence, the user isn't logged out from the IdP in that case.
```
- When ADFS is configured*
- if the ADFS relaying*

tamigoodall committed:
Fixed typo

Fixed typo

tamigoodall committed:
Fixed typos

Fixed typos

nicosabena commented on articles/connections/enterprise/adfs.md:
```diff
@@ -22,45 +22,41 @@ useCase:
+# Connect Your Application to Microsoft ADFS
 
-First, provide this information to your ADFS administrator:
+To connect your application to Microsoft ADFS, you will need to provide the following information to your Active Directory Federation Services (ADFS) administrator:
```
As far as I can tell, MS calls the product "Active Directory Federation Services", without the "Microsoft" or "Windows" words in it. Our dashboard also says only "ADFS".
Maybe "To connect your application to Microsoft's ADFS"?

nicosabena commented on articles/protocols/saml/adfs.md:
```diff
@@ -4,202 +4,137 @@ toc: true
+# Setup a Microsoft ADFS SAML Connection
 
-Auth0 allows you to create a custom <dfn data-key="security-assertion-markup-language">SAML</dfn> connection to Microsoft's Active Directory Federation Services. In addition to getting a bit more flexibility when configuring your mappings, the SAML Connection allows you identity provider-initiated flows (this is something that you cannot do with WS-Fed).
+Create a custom <dfn data-key="security-assertion-markup-language">SAML</dfn> connection to Microsoft's Active Directory Federation Services (ADFS) to get more flexibility when configuring your mappings. The SAML Connection allows you identity provider-initiated flows which you cannot do with WS-Fed.
```
I would change "The SAML Connection allows you identity provider-initiated flows which you cannot do with WS-Fed." to "In addition, SAML connections allow [IdP-Initiated flows](protocols/saml/idp-initiated-sso), which are not supported with regular ADFS-type connections.". Or, considering that we don't recommend IdP-initiated flows, I'd just remove that sentence altogether.

tamigoodall replied:
Removed

nicosabena commented on articles/protocols/saml/adfs.md:
```diff
@@ -4,202 +4,137 @@ toc: true
 4. Return to ADFS and load the downloaded certificate using the **Signatures** tab of the Relying Party properties dialog.
 
-### Map Your Claims
+## Optional: Map additional claims
```
This step is mandatory, not optional. As part of the instructions, you can add a suggested mapping:

   | LDAP Attribute | Outgoing Claim |
    | - | - |
    | E-Mail-Addresses | E-Mail Address |
    | Display-Name | Name |
    | User-Principal-Name | Name ID |
    | Given-Name | Given Name |
    | Surname | Surname |

Also, make a note that the `NameID` outgoing claim should always be present to ensure correct session handling (later on there's a troubleshooting entry for logouts that mentions that this claim is required).

tamigoodall replied:
Updated

nicosabena commented on articles/protocols/saml/saml-configuration/troubleshoot/index.md:
```diff
@@ -51,12 +51,6 @@ When troubleshooting, we recommend beginning by gathering information that helps
-
-* [SAML Troubleshooting When Auth0 is the Identity Provider](/protocols/saml/saml-configuration/troubleshoot/auth0-as-idp)
-
 ## If the Issue Affects Only One (or Just a Few) Users
```
I'm not quite sure about removing the "Troubleshooting approaches". Without them as the page content, this whole page has little substance, and the "Keep reading" list of articles is easily missed IMO.

tamigoodall replied:
Added back in

tamigoodall committed:
Incorporated review comments

Incorporated review comments

nicosabena commented on articles/protocols/saml/adfs.md:
```diff
@@ -4,202 +4,139 @@ toc: true
-![](/media/articles/protocols/saml-adfs/saml16.png)
-
-This launches the Certificate Export Wizard. Click **Next** to proceed.
+1. To get your Federation Metadata, navigate to the following URL:
```
I believe that the whole "Obtain the federation metadata" step can be omitted. The sole purpose is to obtain the callback URL (ACS URL), which is always in the form of `https://{auth0_domain}/login/callback?connection={connection_name}`. This format is explained below, so no need to do this step.

tamigoodall replied:
Deleted

nicosabena commented on articles/protocols/saml/adfs.md:
```diff
@@ -4,202 +4,139 @@ toc: true
+    
+    | Parameter | Example Value |
+    | - | - |
+    | Post-back URL | `https://{yourAuth0accountdomain}/login/callback}/login/callback?connection={your new SAML connection}` |
```
I'd be good to explain either here or below in Trusted URL step of "Edit the Relying Party Trust" that {yourAuth0accountdomain} refers to the custom domain is one is being used.

tamigoodall replied:
Added note

tamigoodall committed:
Update adfs.md

Removed Federation Metadata step (not needed)
Added comment for custom domain post-back URL

tamigoodall committed:
Update adfs.md

Clarified post-back URL
